app/policies/user_policy.rb

```diff
@@ -21,6 +21,10 @@ def update_intent?
     show?
   end
 
+  def view_metric_reports?
```

Comment:
Hmm, interesting point. I'm actually not sure - I could imagine an admin wanting to see a report for _all_ of the students across all the draws, or for all groups across all draws. That said, I definitely stuck it in the `UserPolicy` because I wasn't thinking correctly about how Pundit works, so there's that.

Given that this is general, what do you think about adding it to `ApplicationPolicy`? This is one place where I miss CanCanCan; being able to add arbitrary "permissions" was nice. Let me know what you think!